rfcs/meta_yml_notify.md

```diff
@@ -0,0 +1,50 @@
+# RFC 37: Split META.yml `suggested_reviewers` into `suggested_reviewers` and `notify`
+
+## Summary
+
+Change `META.yml` files in wpt to have `notify` in addition to `suggested_reviewers`.
+@wpt-pr-bot mentions everyone in `notify` and then requests review from one person from `suggested_reviewers`.
+
+## Details
+
+At TPAC 2019 we set out some [priorities for 2020](https://docs.google.com/document/d/1gie7LFb6cAUfabY86MYuWM7m7ux_FaKhDkLdpz0zWkg/edit),
+which include increasing PR review velocity and reduce the number of stale PRs.
+
+wpt has META.yml files with `suggested_reviewers`. @wpt-pr-bot will request review from everyone listed in `suggested_reviewers` for PRs that change that directory, and also assign one of them in a round-robin fashion.
+
+See https://web-platform-tests.org/reviewing-tests/index.html#notifications
+
+The documentation only says that `suggested_reviewers` is for getting notifications for PRs.
+
+This has a few problems:
+
+* There is a disconnect between what contributors would reasonably expect and what often actually happens. When people are requested review and someone is assigned, it communicates that the person assigned will help the contributor to get it landed.
+* The current system doesn't support varying preferences from reviewers. Some people want to see all PRs for a given folder (which is supported by the current system), while others want to help with reviews but not be notified for everything, and some may want to be notified for everything but can't commit to helping with reviews.
+
+To address these problems, have two separate keys in `META.yml`:
+
+* `suggested_reviewers`: @wpt-pr-bot will only request review from one of these, similar to how assignee currently works.
+* `notify`: @wpt-pr-bot will add a comment (before requesting reviews) notifying everyone listed.
+
+Either @wpt-pr-bot can continue to also assign the selected reviewer, or it can only request review and not assign anyone. (This should be decided before merging this RFC.)
+
+This setup allows the following combinations:
+
+* People who want to help with review and want to be notified for everything add themselves to both lists.
+* People who want to help with review but not be notified for everything add themselves to `suggested_reviewers`.
+* People who don't want to help with review but still be notified for everything add themselves to `notify`.
+
+## Risks
+
+Having bots add comments in wpt has been a source of contention in the past. This RFC reintroduces a comment.
```

Comment:
Yeah, I'm worried about this. In general I think if we were going to have a comment from bots, I'd expect much more value in providing a concise summary of test failures, surfacing warnings, or linking to results, than in reverting to a system where we `@`-notify people in a comment in the hope that tweaking notifications makes people spend more time reviewing PRs.

Reply:
The purpose of those comments wouldn't mainly be to get review, but rather:
- move people from `suggested_reviewers` to `notify` if they have no intention of reviewing
- put people in both `suggested_reviewers` and `notify` if they do want to review but also don't want to miss anything. I think @annevk is likely in this category.

Reply:
Sure, but the main concern still applies. If we have some acceptable non-zero number of bot comments we can add without annoying people too much, why is this the most useful thing to spend that budget on? We have a lot of information that's buried several links deep in the checks API that we could instead surface in a comment, for example. To me that seems likely to be higher impact than tweaking the notification system to allow people to be notified without being flagged for review.

Reply:
I don't think having mentions instead of review requests is more important than surfacing other information hidden in CI checks, but I think the problem this RFC is trying to address is worth doing something about.

Personally, I'd love to have a bot comment that surfaces all useful information we want people to see (such as [wptpr.live links](https://github.com/web-platform-tests/wpt/issues/19607)).

Reply:
I don't want bot comments. A bot modifying OP as the PR bot for standards does is fine, but adding comments is noisy and annoying.

Does adding usernames in an edit result in notifications for those added?

Reply:
It does, yes. (Tested in https://github.com/web-platform-tests/wpt/pull/20660 )

Editing OP would be ok for me.

Reply:
I've changed the RFC to say to edit OP.